crates/pathfinder/src/state/block_hash.rs

```diff
@@ -684,4 +712,58 @@ mod tests {
             VerifyResult::Match(_)
         );
     }
+
+    #[test]
+    fn test_transaction_hash_with_signature_0_13_2() {
```

Comment:
Are the test vectors the same as used by starknet-api? If yes, might be worth a comment mentioning the source of these magic numbers.